src/base/bittorrent/sessionimpl.h

```diff
@@ -828,13 +825,10 @@ namespace BitTorrent
         QHash<quint16, std::vector<lt::port_mapping_t>> m_mappedPorts;
 
         QTimer *m_wakeupCheckTimer = nullptr;
-        QDateTime m_wakeupCheckTimestamp;
+        std::chrono::steady_clock::time_point m_wakeupCheckTimestamp;
```

Comment:
> maybe use `QElapsedTimer`?

I would not like to devote even more of my time to this. It's enough for me to get rid of bad performance code. Besides, it looks strange to me to use `QElapsedTimer` in tandem with `QTimer`.

Reply:
I hope you don't mind me change it in another PR? IMO it will fit nicely.

Reply:
> I hope you don't mind me change it in another PR? IMO it will fit nicely.

No problem. I just don't want to continue with it myself, since I have achieved my goal.